src/script/properties/PropertiesRepository.ts

```diff
@@ -205,8 +212,26 @@ export class PropertiesRepository {
       });
   }
 
+  private fetchTypingIndicatorSetting(): Promise<void> {
+    const property = PropertiesRepository.CONFIG.WIRE_TYPING_MODE;
```

Comment:
Change it to smth like that, also You can rename key and default value :)

```suggestion
    const {key, defaultValue} = PropertiesRepository.CONFIG.WIRE_TYPING_MODE;
```

Reply:
I think it's fine to keep it in a `property` object, is makes things clearer when reading the rest of the code. 

I'd rather have `property.key` rather than only `key` which is too generic